nc2zarr/writer.py

```diff
@@ -72,6 +85,17 @@ def __init__(self,
         self._output_append = output_append
         self._output_append_dim = output_append_dim or DEFAULT_OUTPUT_APPEND_DIM_NAME
         self._output_adjust_metadata = output_adjust_metadata
+        if output_append_mode not in _APPEND_MODES:
```

Comment:
This doesn't work, since `output_append_mode` is a `str` and `AppendMode` is an `enum`. This would work:

```python
if output_append_mode in [mode.name for mode in AppendMode]:
```

Alternatively, I could change the function signature to take `output_append_mode` as an `AppendMode` rather than a `str`, and do the string validity checking and conversion to enum in  `Converter` or `cli.nc2zarr`.

Or I could remove this check entirely and just let the `AppendMode(output_append_mode)` in the following line fail, but then the user gets a less helpful error message.

Which would you prefer?

Reply:
> Alternatively, I could change the function signature to take `output_append_mode` as an `AppendMode` rather than a `str`, and do the string validity checking and conversion to enum in `Converter` or `cli.nc2zarr`.

This one. It is the cleanest and clearest.

Reply:
Done! (Commit cd5329b).